index.css

```diff
@@ -0,0 +1,115 @@
+html,
+body {
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+header > * {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.container > * {
+  flex: 1 1 auto;
+}
+
+.single-shape .triangle {
+  background-image: url(static/images/triangle.png);
+  height: 100%;
+  width: 100%;
+  background-repeat: no-repeat;
+}
+
+.single-shape .square {
+  margin-left: -40px;
+  background-image: url(static/images/square.png);
+  height: 100%;
+  width: 100%;
+  background-repeat: no-repeat;
+}
+
+nav .icon {
+  background-image: url(static/images/ic_menu.png);
+  background-repeat: no-repeat;
+  position: absolute;
+  width: 111px;
+  height: 74px;
+  right: -40px;
+  top: 0;
+  margin-top: 30px;
+  justify-self: end;
+  align-self: center;
+}
+
+.single-shape {
+  position: absolute;
+  width: 111px;
+  height: 74px;
+  left: 0;
+  top: 0;
+}
+
+.headline {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 200px;
+}
+
+.headline .about {
+  position: absolute;
+  width: 220px;
+  height: 100px;
+  left: 16px;
+  top: 104px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 50px;
+  color: rgb(99, 99, 241);
+  line-height: 0.9;
+}
+
+.headline button:hover {
+  background-color: green;
+}
+
+.headline .main {
+  position: absolute;
+  width: 300px;
+  height: 126px;
+  left: 16px;
+  top: 224px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 25px;
+  letter-spacing: -0.02em;
+  color: #1a2236;
+  margin-top: 200px;
+  align-self: flex-end;
+  text-align: justify;
+  margin-left: 80px;
+}
```

Comment:
- [ ] [OPTIONAL] I highly recommend that you use the full flexbox feature instead of `position: absolute;`. It will serve you well in the long run when it comes to responsiveness.